### PR TITLE
Quill Orc

### PR DESCRIPTION
**Pull request by MM1953**
Nerdy orcs with excellent magical aptitudes and terrible melee weapon and defensive aptitudes. They read scrolls and memorize spell in half the normal time.

### CODE REVIEW

File: crawl-ref/source/dat/species/quill-orc.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  evocations: 0
+str: 6
+int: 10
+dex: 6
+levelup_stat_frequency: 5
+levelup_stats:
+  - int
+fake_mutations:
+  - long: You read scrolls and books at an uncanny pace.
+  - short: fast scrolls
```
Comment (alexjurkiewicz) [inline crawl-ref/source/dat/species/quill-orc.yaml]: `fast reader` might be more appropriate

File: crawl-ref/source/item-use.cc [hunk 2]
```diff
@@ -2872,14 +2875,21 @@ void read(item_def* scroll)
...
+        const int turns = max(1, you.get_mutation_level(MUT_BLURRY_VISION) - qo - 1);
         start_delay<BlurryScrollDelay>(turns, *scroll);
-        if (you.get_mutation_level(MUT_BLURRY_VISION) == 1)
+        if (you.get_mutation_level(MUT_BLURRY_VISION) - qo == 1)
             you.time_taken /= 2;
     }
+    // quill orcs without blurry vision read scrolls faster
+    else if (you.species == SP_QUILL_ORC && !you.get_mutation_level(MUT_BLURRY_VISION))
+    {
+        you.time_taken /= 2;
```
Comment (alexjurkiewicz) [inline crawl-ref/source/item-use.cc:3091]: does this result in scrolls reading in 0 auts if you read them without blurry vision? Should this use `div_rand_round` instead?

**Review by alexjurkiewicz — CHANGES_REQUESTED** [review]
I like the word quill but I keep thinking of hedgehog rather than pen....

**Comment by alexjurkiewicz** [comment]
I've merged a PR adding a bloatcrawl-specific README, please add the species description there as well

File: crawl-ref/source/item-use.cc (hunk 2 above)
Comment (MM1953) [reply crawl-ref/source/item-use.cc:3091]: I bug-tested this before making the PR, and it results in reading in 0.5 auts, as intended (I think you.time_taken is AUTSx10). I also made sure that it interacts correctly with blurry vision and spell memorization.

**Comment by Hellmonk** [comment]
ready to merge this if someone can resolve the conflicts

**Pull request merged**